neptune/management/internal/api.py

```diff
@@ -81,21 +81,23 @@ def _get_backend_client(api_token: Optional[str] = None) -> SwaggerClientWrapper
 
 
 def get_project_list(api_token: Optional[str] = None) -> List[str]:
-    """Get a list of projects you have access to.
+    """Lists projects that the account has access to.
+
     Args:
-        api_token(str, optional): User’s API token. Defaults to `None`.
-            If `None`, the value of `NEPTUNE_API_TOKEN` environment variable will be taken.
-            .. note::
-                It is strongly recommended to use `NEPTUNE_API_TOKEN` environment variable rather than placing your
-                API token in plain text in your source code.
+        api_token: Account's API token.
+            If None, the value of NEPTUNE_API_TOKEN environment variable will be taken.
+            Note: To keep your token secure, use the NEPTUNE_API_TOKEN environment variable rather than placing your
+            API token in plain text in your source code.
+
     Returns:
-        ``List[str]``: list of project names of a format 'WORKSPACE/PROJECT'
-    Examples:
+        List of project names in the form 'workspace-name/project-name'.
+
+    Example:
         >>> from neptune import management
         >>> management.get_project_list()
-    You may also want to check `management API reference`_.
-    .. _management API reference:
-       https://docs.neptune.ai/api-reference/management
+
+    You may also want to check the management API reference:
+    https://docs.neptune.ai/api-reference/management
```

Comment:
maybe better link would be 'https://docs.neptune.ai/api-reference/management#.get_project_list' ??  
What do you think?

Reply:
You're right that many of these links could be more specific. The reason I slapped the same link everywhere is that the URLs will change with the release of the new docs, so I allowed myself to cut this corner...